extensions/http-curl/tests/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ message("-- Finished building ${CURL_INT_TEST_COUNT} libcURL integration test fi
 
 add_test(NAME HTTPClientTests COMMAND "HTTPClientTests" WORKING_DIRECTORY ${TEST_DIR})
 add_test(NAME HTTPStreamingCallbackTests COMMAND "HTTPStreamingCallbackTests" WORKING_DIRECTORY ${TEST_DIR})
+add_test(NAME InvokeHTTPTests COMMAND "InvokeHTTPTests" WORKING_DIRECTORY ${TEST_DIR})
```

Comment:
a few lines above, we have a loop iterating over the unit tests, (`HTTPClientTests`, `HTTPStreamingCallbackTests`, `InvokeHTTPTests`) and there is a commented out `add_test`, should we uncomment that instead and remove these three here?

Reply:
Good catch! Updated in 5291683b4341a0a5ae787785458df8b414cb0cda